src/parser/jobs/ast/modules/ArcanaTracking/ArcanaSuggestions.tsx

```diff
@@ -200,7 +212,7 @@ export default class ArcanaSuggestions extends Module {
 
 	// Helper for output()
 	RenderSpreadState(artifact: CardLog) {
-		const drawnArcana = getDataBy(STATUSES, 'id', artifact.drawState)
+		const drawnArcana = artifact.drawState ? getDataBy(this.data.statuses, 'id', artifact.drawState) : undefined
```

Comment:
You can use `this.data.getStatus(artifact.drawState)` here, since it's gonna be either the status or `undefined` due to the `getStatus` sig anyway.

Reply:
Good catch